src/main/resources/db/migration/V212__mce_datafixes.sql

```diff
@@ -0,0 +1,179 @@
+WHERE court_id = 1480145;
+
+UPDATE search_servicecentre ss
+SET intro_paragraph_cy = 'Mae''r lleoliad hwn yn gwasanaethu Cymru a Lloegr gyfan ar gyfer cyfryngu hawliadau bychain ar y rhan fwyaf o achosion o hawliadau bychain a amddiffynnwyd. Mae''r gwasanaeth yn rhad ac am ddim ac yn helpu i ddatrys anghydfodau ariannol heb fod angen gwrandawiad llys. Gallwch gael rhagor o wybodaeth yn y canllaw ar gyfer (https://www.gov.uk/guidance/small-claims-mediation-service) Gwasanaeth Cyfryngu Hawliadau Bychain ar GOV.UK. Nid ydym yn darparu gwasanaeth wyneb yn wyneb.'
+WHERE court_id = 1480145;
+
+
+
+UPDATE search_facility ss
+SET description  = REGEXP_REPLACE(description, '<[^>]+>', '', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '<[^>]+>', '', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&nbsp;', ' ', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&nbsp;', ' ', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&amp;', '&', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&amp;', '&', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&#39;', '''', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&#39;', '''', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&ndash;', '–', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&ndash;', '–', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&lsquo;', '''', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&lsquo;', '''', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&eacute;', 'é', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&eacute;', 'é', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&middot;', '·', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&middot;', '·', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&#64;', '@', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&#64;', '@', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&rsquo;', '''', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&rsquo;', '''', 'g');
+
+UPDATE search_facility ss
+SET description = REGEXP_REPLACE(description, '&acirc;', 'â', 'g');
+UPDATE search_facility ss
+SET description_cy = REGEXP_REPLACE(description_cy, '&acirc;', 'â', 'g');
```

Comment:
A quick one. In addition to the above, lets say we have <a href="blabla">click here</a>

I believe the above would remove the element tags but would keep "click here". Do correct me if I am wrong. Not the end of the world I guess, but better to rectify now while we can

Reply:
There's no cases like click here in the databases i checked all the distinct variations of strings that came up